source/_components/device_tracker.owntracks_http.markdown

```diff
@@ -21,22 +21,40 @@ To integrate Owntracks tracking via HTTP in Home Assistant, add the following se
 # Example configuration.yaml entry
 device_tracker:
   - platform: owntracks_http
+    webhook_id: long-random-webhook-url
 ```
 
-For configuration options and usage instructions, read the documentation for the [OwnTracks platform](/components/device_tracker.owntracks/).
+## IMPORTANT!!!
```

Comment:
We have a notification box for that. See our developers documentation in case you are still interested to display a warning or informational message box.

Reply:
I did look at https://developers.home-assistant.io but didn't find anything like that in the Documentation section. Let's leave it as it is.